templates/prometheusproxynetworkpolicy.go

```diff
@@ -34,7 +32,6 @@ var PrometheusProxyNetworkPolicyTemplate = netv1.NetworkPolicy{
 			{
 				Ports: []netv1.NetworkPolicyPort{
 					{
-						Port:     &prometheusProxyPort,
```

Comment:
This is done to enable prometheus to query redhat sso and get a response from the callback url on all port since we are not sure which port it will create the connection from,
But yes  will check if it affects federation.

Reply:
Doesn't affect fedreation, but it does open all ports on prometheus. Trying to restrict that

Reply:
Checked up again and verified on a running cluster, prometheus proxy network port not causing issues connecting.
Might have been something else when I had initially tested it.
Reverting change.